wireapp/wire-android
# fix: request video steam for screen share (AR-3132) (RC)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3132" title="AR-3132" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />AR-3132</a>  Not receiving screen sharing
  </td></table>
  <br />
 




----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

We don't receive screen sharing in AR.

### Causes (Optional)

`requestVideoStream` usecase is not called when someone starts sharing his screen.

### Solutions

Invoke `requestVideoStream` usecase for screen sharing as well

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

In a call try to share your screen from web/desktop
You should be able to see that video stream on your Android app

### Notes (Optional)

Sister PR of #1454 

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 



----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
